examples/build.gradle

```diff
@@ -1,5 +1,5 @@
-project.ext.sdkVersion = 23
-project.ext.buildTools = '23.0.0'
+project.ext.sdkVersion = 24
```

Comment:
Because most devices cannot support SDK version 24 yet, I don't know that using SDK version 23 is a good idea.